src/cascadia/TerminalControl/XamlLights.idl

```diff
@@ -1,13 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+#define XAML_LIGHT(Name) \

```

Comment:
Can we just define a base class like

```
runtimeclass XamlLight {
    static Windows.UI.Xaml.DependencyProperty IsTargetProperty { get; }; 
    static Boolean GetIsTarget(Windows.UI.Xaml.DependencyObject target); 
    static void SetIsTarget(Windows.UI.Xaml.DependencyObject target, Boolean value);
}
```

that `VisualBellLight` and `CursorLight` both implement instead? Doing this with a macro does seem like a weird pattern...

Reply:
Or is it because these are all static functions, that have to be directly declared by the class that they're defined for. That would be annoying.

Reply:
Yeah there seems to be some issue with defining a runtimeclass that _only_ has static functions